docs/contributor/benchmarks/README.md

```diff
@@ -335,21 +335,25 @@ test results are printed out.
+|                      3.1.6 |                  9423                   |                  30652                   |              37562              |         105          |      0.9      |                  4522                   |                   9517                   |             2414417             |         631          |      0.4      |                   4002                   |                   4945                   |             1897270             |         693          |      0.5      |                   3224                   |                   1680                   |             4898631             |         634          |      0.3      |
+| 3.1.3 (after optimization) |                  9149                   |                  29694                   |              37883              |        93, 98        |    1, 0.8     |                  4344                   |                   8354                   |             2893257             |       601, 587       |   0.4, 0.4    |                   3715                   |                   4000                   |             1947742             |       739, 692       |   0.4, 0.4    |                   3200                   |                   1314                   |             4950684             |       662, 693       |   0.3, 0.3    |
+| 3.1.6 (after optimization) |                  10183                  |                  33117                   |              30358              |        98, 93        |     1, 1      |                  4349                   |                   8935                   |             2779129             |       539, 660       |   0.4, 0.4    |                   5471                   |                   5840                   |             1889836             |       646, 641       |   0.5, 0.6    |                   3224                   |                   1621                   |             4882502             |       686, 649       |   0.3, 0.3    |
 </div>
 
+> **NOTE:** The test results can vary up to 1000KByte/sec in the input, and up to 3000KByte/sec in the output processing rate with single pipeline tests. The Multi Pipeline test results can be lower than the single pipeline test results, as the test setup is shared between multiple pipelines and this may create pressure on the shared resources.
```

Comment:
```suggestion
> **NOTE:** The test results can vary up to 1000KByte/sec in the input, and up to 3000KByte/sec in the output processing rate with single pipeline tests. The multi pipeline test results can be lower than the single pipeline test results, because the test setup is shared between multiple pipelines and this may create pressure on the shared resources.
```